messages/perl/dist.ini

```diff
@@ -29,8 +29,13 @@ repository.type   = git
 [PkgVersion]
 [Prereqs::FromCPANfile]
 [Git::GatherDir]
-exclude_filename=VERSION
 exclude_filename=default.mk
+exclude_filename=Makefile
+exclude_filename=VERSION
+
+[GatherFile]
+; explicitly add unversioned files
```

Comment:
CHANGELOG.md should be versioned 🤔
Isn't it?

Reply:
It *was* versioned in the `messages/perl/` directory, but after the Cucumber::Messages 17.0.1 release, it was removed as a versioned file. The release procedure copies it into the module directory as an unversioned file. Same procedure is used for the `gherkin/perl/` directory. (Where Gherkin 21.0.0 doesn't have a CHANGELOG.md file in the distribution archive because it was unversioned at the time of the release.)

Reply:
Ok, got it, thanks